ilugd.py

```diff
@@ -82,17 +82,25 @@ def help(bot, update):
 /github - link to ilugd github repos
 ''')
 
+def unknown(bot, update):
+    bot.sendChatAction(chat_id=update.message.chat_id,
+                       action=ChatAction.TYPING)
+    bot.sendMessage(chat_id=update.message.chat_id, text="Unknown command. Use /help")
 
 # Welcome a user to the chat
 def welcome(bot, update):
     message = update.message
     chat_id = message.chat.id
-    phrases = ['Hello {}! Welcome to {} .Please introduce yourself. Dont forget to read the chat rules using !rules command'.format(message.new_chat_member.first_name,message.chat.title),
-               'Hi {}! Welcome to {} .let\'s start with introduction. Dont forget to read the chat rules using !rules command'.format(message.new_chat_member.first_name,message.chat.title)
-               #'Hello {}! Welcome to {} .Please introduce yourself.'.format(message.new_chat_member.first_name,message.chat.title),
-               #'Hello {}! Welcome to {} .Please introduce yourself.'.format(message.new_chat_member.first_name,message.chat.title),
-               #'Hello {}! Welcome to {} .Please introduce yourself.'.format(message.new_chat_member.first_name,message.chat.title)
-                ]
+
+    try:
+        # new_chat_members return a list of dicts from which member's username has to be searched
+        member_first_name = [member['first_name'] for member in message.new_chat_members if 'first_name' in member]
+    except KeyError:
+        member_first_name = message.new_chat_members[0]['first_name']
+		
+    phrases = ['Hello {}! Welcome to {} .Please introduce yourself. Dont forget to read the chat rules using !rules command'.format(member_first_name,message.chat.title)]
```

Comment:
why modify the list it was planned to expand it so that if multiple people join on same day its too boring to receive same message

Reply:
There was no documentation or comment about it. I will write a comment and add a few more welcoming messages. Should I proceed?

Reply:
applies to same sticker what do you think (i was thinking like a list and throw a random one to chat)

Reply:
> There was no documentation or comment about it. I will write a comment and add a few more welcoming messages. Should I proceed?

yupz (also for stickers (I love gifs))

Reply:
It sounds awesome! Also what about the '!rules' command mentioned in 'phrases'? How should I go about it?

Reply:
ping @nk521 for more info (@nk521 we can merge both ?)
it's from a different bot

Reply:
So, for the time being, I will make changes for stickers and 'phrases' and will raise an issue about the 'rules' command to be solved in future commits. Is that all right?